packages/vue/src/components/IonApp.ts

```diff
@@ -16,6 +16,8 @@ export const IonApp = /*@__PURE__*/ defineComponent((_, { attrs, slots }) => {
   };
 });
 
+IonApp.name = "IonApp";
```

Comment:
I noticed that other components set the `name` as part of the initial `defineComponent` call: https://github.com/ionic-team/ionic-framework/blob/main/packages/vue/src/components/IonPage.ts#L4 Is it possible to do that here, or is there a blocking difference in how the components are set up? I'm just wary of this line being missed if we have to copy-paste this setup in the future. (Ditto for `IonBackButton` and `IonNav`)

Reply:
Gotcha, thanks 👍